File: solutions/beecrowd/2936/2936.cpp
#include <cstdint>
#include <iostream>

int main() {
    int16_t curupira, boitata, boto, mapinguari, iara;

    while (std::cin >> curupira >> boitata >> boto >> mapinguari >> iara) {
        std::cout << curupira * 300 + boitata * 1500 + boto * 600
                + mapinguari * 1000 + iara * 150 + 225
                  << std::endl;
    }

    return 0;
}

File: solutions/beecrowd/2936/2936.py
import sys

input = sys.stdin.readline
output = sys.stdout.write

while line := input().strip():
    curupira = int(line)
    boitata = int(input())
    boto = int(input())
    mapinguari = int(input())
    iara = int(input())

    result = (
        curupira * 300
        + boitata * 1500
        + boto * 600
        + mapinguari * 1000
        + iara * 150
        + 225
    )
    output(f'{result}\n')
